### PR TITLE
REVIEW: Program crashes when modifying book that is not in database

### PR DESCRIPTION
**Pull request by wyatt-geiger**
This is Issue #8: "Program crashes when modifying book that is not in database"

With this validation the program will no longer crash if the book ID is not in the database. An error message is displayed and the program then displays the main menu to the user.

@davidyunda @DashKazak

### CODE REVIEW

File: bookstore.py [hunk 1]
```diff
@@ -192,11 +192,13 @@ def get_book_by_id(self, id):
             
             if book_data:
                 book = Book(book_data['title'], book_data['author'], book_data['read'], book_data['rowid'])
-                    
-            con.close()            
-            
-            return book 
+            else:
+                book = None # Wyatt: If the book_data isn't found in the database, book is set to None
```
Comment (claraj) [inline bookstore.py]: You don't need your name here. Git can report who wrote any line of code in the project.

File: main.py [hunk 2]
```diff
@@ -61,9 +61,13 @@ def change_read():
 
     book_id = ui.get_book_id()
     book = store.get_book_by_id(book_id)  
-    new_read = ui.get_read_value()     
-    book.read = new_read 
-    book.save()
+
+    if book != None: # Wyatt: If book value is not None, then the changes are saved
```
Comment (claraj) [inline main.py]: Python style suggests

`if book is not None:`

rather than 

`if book != None:`

but it's correct either way.

**Comment by claraj** [comment]
FYI who wrote what line of code? Git blame, look for the blame button on any page of code, for example https://github.com/DashKazak/readinglist/blame/book_not_in_db/main.py
image

**Review by DashKazak — APPROVED** [review]
LGTM

**Pull request merged**